### PR TITLE
Bugfix and format signup and login

### PR DESCRIPTION
**Pull request by jackhallam**

### CODE REVIEW

File: src/main/java/edu/rosehulman/quota/controller/FreezeUserController.java [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
-
 import edu.rosehulman.quota.Database;
 import edu.rosehulman.quota.model.User;
 import spark.Request;
 import spark.Response;
 import spark.Route;
 
+import java.util.Optional;
```
Comment (poe9695) [inline src/main/java/edu/rosehulman/quota/controller/FreezeUserController.java:11]: What's the point of reorder? Or is this just the formatting...?

File: src/main/java/edu/rosehulman/quota/controller/SetConfigController.java [hunk 2]
```diff
@@ -33,19 +32,13 @@ public Object handle(Request request, Response response) throws Exception {
...
-    partner.setApiKey(apiKey);
 
-    addPartnerToDatabase(partner);
+    String partnerId = partnerJsonObject.get("partnerId").getAsString();
 
     JsonArray productsJsonArray = partnerJsonObject.getAsJsonArray("products");
     productsJsonArray.iterator().forEachRemaining(productJsonElement -> {
       JsonObject productJsonObject = productJsonElement.getAsJsonObject();
-      String productId = productJsonObject.get("id").getAsString();
+      String productId = productJsonObject.get("productId").getAsString();
```
Comment (poe9695) [inline src/main/java/edu/rosehulman/quota/controller/SetConfigController.java:41]: Are all the Id changes from the new format of the config?

File: src/main/java/edu/rosehulman/quota/controller/SetConfigController.java [hunk 3]
```diff
@@ -58,7 +51,7 @@ public Object handle(Request request, Response response) throws Exception {
       JsonArray quotasJsonArray = productJsonObject.getAsJsonArray("quotas");
       quotasJsonArray.iterator().forEachRemaining(quotaJsonElement -> {
         JsonObject quotaJsonObject = quotaJsonElement.getAsJsonObject();
-        String quotaId = quotaJsonObject.get("id").getAsString();
+        String quotaId = quotaJsonObject.get("quotaId").getAsString();
```
Comment (poe9695) [inline src/main/java/edu/rosehulman/quota/controller/SetConfigController.java:54]: ditto

File: src/main/java/edu/rosehulman/quota/controller/SetConfigController.java [hunk 4]
```diff
@@ -74,10 +67,9 @@ public Object handle(Request request, Response response) throws Exception {
         JsonArray tiersJsonArray = quotaJsonObject.getAsJsonArray("tiers");
         tiersJsonArray.iterator().forEachRemaining(tierJsonElement -> {
           JsonObject tierJsonObject = tierJsonElement.getAsJsonObject();
-          String tierId = tierJsonObject.get("id").getAsString();
```
Comment (poe9695) [inline src/main/java/edu/rosehulman/quota/controller/SetConfigController.java:77]: ditto

**Review by poe9695 — APPROVED** [review]

File: src/main/java/edu/rosehulman/quota/controller/FreezeUserController.java (hunk 1 above)
Comment (jackhallam) [reply src/main/java/edu/rosehulman/quota/controller/FreezeUserController.java:11]: That was intellij reordering them

File: src/main/java/edu/rosehulman/quota/controller/SetConfigController.java (hunk 2 above)
Comment (jackhallam) [reply src/main/java/edu/rosehulman/quota/controller/SetConfigController.java:41]: This was a bug, `id` is *shared's* id for the product, not the actual product id. We need to use `productId` to get the actual id.

**Pull request merged**